### DISCUSSION TITLE
Make the operationName parameter optional when fetching graphql schema

### DISCUSSION THREAD

**Discussion by MENGKONGSEAN** [Kong/insomnia#3770] [Ideas]
**Describe the bug**
when I click fetch schema graphql it occurs error because of query we not support `"operationName":"IntrospectionQuery"`

**To Reproduce**
Just fetch from in graphql

**Expected behavior**
Should retrieve data.
**Screenshots**
![image](https://user-images.githubusercontent.com/49071018/106722966-e0a9e680-6606-11eb-9c5e-986cc22e0649.png)

**Desktop (please complete the following information):**
 - OS: [arch linux]
 - App Version [7.1.1]
![image](https://user-images.githubusercontent.com/49071018/106723070-046d2c80-6607-11eb-8db3-c38e761139d6.png)

**Additional context**
how to remove `"operationName":"IntrospectionQuery"` from query fetch schema?

**Comment by nijikokun** [comment]
Welcome @MENGKONGSEAN, seems like it would have to be a flag under preferences to disable this [additional field](https://github.com/Kong/insomnia/blob/c4babfc129c737eb46f1f92e2992c72dfae34bfa/packages/insomnia-app/app/ui/components/editors/body/graph-ql-editor.js#L263).

**Comment by wdawson** [comment]
Found this potentially relevant discussion: https://github.com/dotansimha/graphql-code-generator/discussions/4408
